src/components/metadata/metadata.js

```diff
@@ -82,9 +82,10 @@ const MetaData = ({
                   className="pipeline-metadata__icon"
                   icon={nodeTypeIcon}
                 />
-                <h2 className="pipeline-metadata__title">
-                  {metadata.node.name}
-                </h2>
+                <span
+                  className="pipeline-metadata__title"
+                  dangerouslySetInnerHTML={{ __html: metadata.node.name }}
+                />
```

Comment:
Looks like this html element changed from an `h2` to a `span`. It should stay as a `h2`.